optuna/integration/__init__.py

```diff
@@ -129,12 +131,15 @@ def _get_module(self, module_name: str) -> ModuleType:
             try:
                 return importlib.import_module("." + module_name, self.__name__)
             except ModuleNotFoundError:
-                raise ModuleNotFoundError(
-                    "Optuna's integration modules for third-party libraries have started "
-                    "migrating from Optuna itself to a package called `optuna-integration`. "
-                    "The module you are trying to use has already been migrated to "
-                    "`optuna-integration`. Please install the package by running "
-                    "`pip install optuna-integration`."
-                )
+                is_module_deprecated = module_name in ("allennlp", "chainer", "chainermn")
```

Comment:
How about adding all the options to `pyproject.toml` in `optuna-integration`? I think no exceptions would lower management costs.

Reply:
I applied your suggestion along with `optuna-integration`:
- https://github.com/optuna/optuna-integration/pull/130